features/environment.py

```diff
@@ -342,22 +335,34 @@ def before_all(context):
         if context.config.userdata.get('start-storage-service'):
             context.PROCESSES['commissaire-storage-service'] = try_start(
                 start_commissaire_service, 'commissaire-storage-service',
-                context, ['commissaire-storage-service'])
+                context, [
```

Comment:
I think this service will need to copy the default and update the storage handler to point to the generated etcd port.